src/main/java/Jarvis/TaskList.java

```diff
@@ -0,0 +1,105 @@
+package Jarvis;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Class representing a list of tasks.
+ * <p>
+ *     This class encapsulates the list of tasks and provides methods for adding, removing, and retrieving tasks.
+ * </p>
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
```

Comment:
Perhaps a more appropriate name for this variable could be "tasks" to highlight that it stores a collection?